[Bug]: yarn build error

C:\CodeProject\teamsai\teams-ai\js>yarn build
 | src/TaskModules.ts(188,44): error TS2454: Variable 'response' is used before being assigned.
error Command failed with exit code 1.

**\teams-ai\js\packages\teams-ai\src\TaskModules.ts**

There is exception in line 188, the response variable is not in closed loop with 

```
   let response: TaskModuleResponse;    
   if (typeof result == 'string') {
   } else if (typeof result == 'object') {
   }
// no closed conditional loop for accessing response below
      await context.sendActivity({
                              value: { body: response, status: 200 } as InvokeResponse,
                              type: ActivityTypes.InvokeResponse
                          }); 

```

--- a/js/packages/teams-ai/src/TaskModules.ts
+++ b/js/packages/teams-ai/src/TaskModules.ts
@@ -165,7 +165,7 @@ export class TaskModules<TState extends TurnState> {
                     const result = await handler(context, state, context.activity.value?.data ?? {});
                     if (!context.turnState.get(INVOKE_RESPONSE_KEY)) {
                         // Format invoke response
-                        let response: TaskModuleResponse;
+                        let response: TaskModuleResponse | undefined = undefined;
                         if (typeof result == 'string') {
                             // Return message
                             response = {

